packages/livecd/grub2/build.yaml

```diff
@@ -1,7 +1,14 @@
+{{ if .Values.arch }}
+  {{if and (eq .Values.arch "aarch64") (eq .Values.distribution "opensuse") }}
+image: {{ .Values.live_image }}
+  {{else}}
```

Comment:
I'm wondering.. wouldn't this apply for teal too? Shouldn't be values adapted there too?

Reply:
Yes indeed, I wanted to see if this built as expected before bringing it tomorrow morning in the meeting